features/org.jboss.tools.arquillian.feature/feature.xml

```diff
@@ -19,15 +19,15 @@
    </license>
 
    <requires>
-      <import feature="org.eclipse.sapphire" version="8.0.0" match="greaterOrEqual"/>
-      <import feature="org.eclipse.sapphire.java" version="8.0.0" match="greaterOrEqual"/>
-      <import feature="org.eclipse.sapphire.java.jdt" version="8.0.0" match="greaterOrEqual"/>
-      <import feature="org.eclipse.sapphire.modeling.xml" version="8.0.0" match="greaterOrEqual"/>
-      <import feature="org.eclipse.sapphire.osgi" version="8.0.0" match="greaterOrEqual"/>
-      <import feature="org.eclipse.sapphire.platform" version="8.0.0" match="greaterOrEqual"/>
-      <import feature="org.eclipse.sapphire.ui.swt.xml.editor" version="8.0.0" match="greaterOrEqual"/>
-      <import feature="org.eclipse.sapphire.ui" version="8.0.0" match="greaterOrEqual"/>
-      <import feature="org.eclipse.sapphire.ui.swt.gef" version="8.0.0" match="greaterOrEqual"/>
+      <import feature="org.eclipse.sapphire" version="9.0.0" match="greaterOrEqual"/>
```

Comment:
Sapphire changed its API in 9.0.0.201505051659 (included in Mars M7 update site). So any previous 9.0.0.* version (Mars M1-M6) won't work with this PR.
We should require 9.0.0.201505051659 (greaterOrEqual) here in the feature.xml and in all corresponding plugins.xml's.